content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx

```diff
@@ -445,6 +445,14 @@ model Category {
   name    String
   postIDs String[] @db.ObjectId
   posts   Post[]   @relation(fields: [postIDs], references: [id])
+  Post    Post[]
+}
+
+enum Sample {
```

Comment:
Something odd's been added here

Reply:
I _think_ I fixed this with my [suggestion](https://github.com/prisma/docs/pull/3611#pullrequestreview-1105838055)

Reply:
Looks like there's still a deleted `}`?